components/role-mgt/org.wso2.carbon.identity.role.mgt.core/src/main/java/org/wso2/carbon/identity/role/mgt/core/IdentityRoleManagementException.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.role.mgt.core;
+
+/**
+ * IdentityRoleManagementException.
+ */
+public class IdentityRoleManagementException extends Exception {
```

Comment:
shall we properly format this class?

Reply:
Can you please elaborate more on the issue you see in the formatting?

Reply:
new lines at https://github.com/wso2/carbon-identity-framework/pull/2947/files/9c35a360ca7261e59f458516e362425d8607e277#diff-1c8709a3911eea2bdf5f91e354c899b4R36 , https://github.com/wso2/carbon-identity-framework/pull/2947/files/9c35a360ca7261e59f458516e362425d8607e277#diff-1c8709a3911eea2bdf5f91e354c899b4R47 , https://github.com/wso2/carbon-identity-framework/pull/2947/files/9c35a360ca7261e59f458516e362425d8607e277#diff-1c8709a3911eea2bdf5f91e354c899b4R78

Reply:
adding new lines after the method declaration.

Reply:
Fixed and updated the PR.